app/helpers/application_helper.rb

```diff
@@ -335,4 +335,124 @@ def speaker_width(rooms)
     # speaker picture padding: 4px 2px; and we want the picture to be a circle
     speaker_height(rooms) - 4
   end
+
+  # Recieves a PaperTrail::Version object
+  # Outputs the list of attributes that were changed in the version (ignoring changes from one blank value to another)
+  # Eg: If version.changeset = '{"title"=>[nil, "Premium"], "description"=>[nil, "Premium = Super cool"], "conference_id"=>[nil, 3]}'
+  # Output will be 'title, description and conference'
+  def updated_attributes(version)
+    version.changeset.
+      reject{ |_, values| values[0].blank? && values[1].blank? }.
+      keys.map{ |key| key.gsub('_id', '').tr('_', ' ')}.join(', ').
+      reverse.sub(',', ' dna ').reverse
+  end
+
+  def change_creator_link(user_id)
+    if user_id
```

Comment:
It is not enough to check if a user_id was given, we need to double check the user exists before displaying a link which will otherwise fail.